spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/sending-messages.adoc

```diff
@@ -650,6 +650,13 @@ public Message<?> messageReturn(String in) {
 }
 ----
 
+[[record-key-in-reply]]
+== Preserving Request Record Key in Reply
```

Comment:
Why do we need a dedicated section for this single short paragraph?
Why it cannot be included in the end if the previous one?
At least it looks like both of them talks about similar context.
Where that `Message` indeed covers what you try to say about multi-value reply.